### PR TITLE
Added more sample docs

### PR DESCRIPTION
**Pull request by hunterwilliams**
If someone could verify the command that would be great. It should be correct but was having issues getting mlcp to work on my project properly.. #159

### CODE REVIEW

File: app/templates/README.mdown [hunk 1]
```diff
@@ -71,26 +71,26 @@ Next, push all source files to the appropriate server. The following assumes it
...
 
 ## Sample Data
 
-The application comes with 25 JSON documents generated by json-generator.com.
+The application comes with 1300 JSON documents generated by json-generator.com.
 You can load them with
 (MLCP)[https://docs.marklogic.com/guide/ingestion/content-pump] like this:
 
-    ./ml local mlcp import -input_file_path sample-data -document_type json
+    ./ml local mlcp import -input_file_path sample-data -document_type json -input_compressed true
```
Comment (grtjn) [inline app/templates/README.mdown]: While in here, can you make some more changes to this line?

- You don't need the `-document_type` param, MarkLogic will auto-detect from extension.
- You can save yourself typing `true`, just `-input_compressed` is enough.
- Could you please add `-output_collections data`, both a good practice, and useful for changing the additional-query to a better default
- Last but not least, we should apply doc-permissions to allow the app-user that gets generated automatically access as well: `output_permissions <app-name>-user:read,<app-name>-user:update`

Oooh, we can automate the <app-name> replacement. Put the above options in an `import-sample-data.options` file, and use @ml.app-name inside there! :)

Comment (grtjn) [inline app/templates/README.mdown]: Regarding output_collections, see also #136..

**Comment by hunterwilliams** [comment]
I'll tackle both at once. Sorta confused on the import-sample-data.options. Will that automatically be modified during the generation process and swap the app name to what we want? Also I'm assuming roxy or mlcp can use the options file?

**Comment by grtjn** [comment]
Roxy does some magic to options files. It reads them, resolves those @ml. patterns, and appends the contents to the system-call it makes to run MLCP..

**Comment by hunterwilliams** [comment]
So the file should read like this?? 

``
local mlcp import -input_file_path sample-data.zip -input_compressed -output_collections data -output_permissions @ml.app-name-user:read,@ml.app-name-user:update
``

Or does <@ml.app-name>-user ? Not sure how roxy does the replace

**Comment by grtjn** [comment]
Close, but MLCP option files should have each argument on a separate line. The MLCP call would read:

```
./ml local mlcp -options_file import-sample-data.options
```

And the options file could contain:
```
import

-input_file_path
sample-data/

-input_compressed

-output_collections
data,data/people

-output_permissions
@ml.app-name-user:read,@ml.app-name-user:update
```

That should do it..

**Comment by hunterwilliams** [comment]
I've added that. Think the search options change should be a separate PR actually. Similar documents doesn't read from search options and going to have to change creating documents to get the collection as well

**Comment by grtjn** [comment]
Yep, separate PR for search options (and similar) makes sense. Good point about similar. Can you create a ticket for that?

**Comment by grtjn** [comment]
PS: changes look great, haven't actually tested yet though..

**Comment by ml** [comment]
Haha, whenever you use @ml, I get a notification ;) I guess I have to ignore the repo ...

**Comment by grtjn** [comment]
Yes, I noticed. I'll try to use back-quotes around it as much as possible.. :)

**Comment by grtjn** [comment]
Almost there.. `@ml.default-user` is both wrong, and does not work properly. Roxy mistakenly looks for `-option_file` instead of `-options_file`. It also inconveniently looks for it in the `deploy/` directory. And permissions work on roles, not users. So it should have been `@ml.app-role`..

Perhaps just put a placeholder there, and ask people in the README to edit the options..

**Comment by grtjn** [comment]
Last but not least, I got the syntax wrong, last line should use just commas, no colons:

```
$app-role,read,$app-role,update
```

**Comment by grtjn** [comment]
Sorry, not last. Not that I mind too much, but there are 3000 docs in the zip, not 1300. And can you also append this to the options file, to strip away the zip path from the uris:

```
-output_uri_replace
".*\.zip,''"
```
Note, that are two single quotes, in double ones..

**Comment by grtjn** [comment]
Otherwise import working nicely, so after those fixes this should be good to go..

**Comment by hunterwilliams** [comment]
All changes should be in now

**Comment by hunterwilliams** [comment]
@grtjn see if this satisfies #169

**Comment by grtjn** [comment]
Nice, getting 7 eyeColors. Definitely better. Now the show more has a purpose too.. ;)

**Comment by grtjn** [comment]
I think I ran sufficient tests with this, merging..

**Pull request merged**